_search-plugins/knn/knn-vector-quantization.md

```diff
@@ -157,3 +163,45 @@ As an example, assume that you have 1 million vectors with a dimension of 256 an
 1.1 * (((2 * 256) * 1,000,000) + (4 * 128 * 256))  ~= 0.525 GB
 ```
 
+## Faiss product quantization
+
+Product quantization is a technique that allows users to represent a vector in a configurable amount of bits. In general, it can be used to achieve a higher level of compression compared to byte and scalar quantization. Product quantization works by breaking up vectors into _m_ subvectors, and  encoding each subvector with _code_size_ bits. Thus, the total amount of memory for the vector ends up being `m*code_size` bits, plus overhead. For more details about the parameters of product quantization, see [PQ parameters]({{site.url}}{{site.baseurl}}/search-plugins/knn/knn-index/#pq-parameters). Product quantization is only supported for the _Faiss_ engine and can be used with either the _HNSW_ or the _IVF_ ANN algorithms.
+
+### Using Faiss product quantization
+
+In order to minimize the loss in accuracy, product quantization requires a _training_ step that builds a model based on the distribution of the data that will be searched over.
+
+Under the hood, the product quantizer is trained by running k-Means clustering on a set of training vectors for each sub-vector space and extracts the centroids to be used for the encoding. The training vectors can either be a subset of the vectors to be ingested, or vectors that have the same distribution and dimension as the vectors to be ingested.
```

Comment:
Avoid using jargon.

Reply:
What do you mean by jargon?

Reply:
For example, "under the hood" is jargon. Please see the OpenSearch writing style guide about tone (https://github.com/opensearch-project/documentation-website/blob/main/STYLE_GUIDE.md#tone). To move this forward, I've accepted writing and editorial changes based on the style guide. I'll move this forward into editorial and once approved get the PR merged. Thank you for contributing to the documentation :)